tests/Tests/500__jupyterhub/culler.robot

```diff
@@ -0,0 +1,169 @@
+*** Settings ***
+Documentation    Tests for the NB culler
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library          ../../../libs/Helpers.py
+Library          OpenShiftLibrary
+Suite Teardown   Teardown
+Force Tags       JupyterHub
+
+
+*** Variables ***
+${DEFAULT_CULLER_TIMEOUT} =    31536000
+${CUSTOM_CULLER_TIMEOUT} =     600
+
+
+*** Test Cases ***
+Verify Default Culler Timeout
+    [Documentation]    Checks default culler timeout
+    [Tags]    Sanity    Tier1
+    ...       ODS-1255
+    Disable Culler
+    ${current_timeout} =  Get Culler Timeout
+    Should Be Equal  ${DEFAULT_CULLER_TIMEOUT}  ${current_timeout}
+    Close Browser
+
+Verify Culler Timeout Can Be Updated
+    [Documentation]    Verifies culler timeout can be updated
+    [Tags]    Sanity    Tier1
+    ...       ODS-1231
+    Modify Culler Timeout    ${CUSTOM_CULLER_TIMEOUT}
+    ${current_timeout} =  Get Culler Timeout
+    Should Not Be Equal  ${current_timeout}  ${DEFAULT_CULLER_TIMEOUT}
+    Should Be Equal   ${current_timeout}  ${CUSTOM_CULLER_TIMEOUT}
+    Close Browser
+
+Verify Culler Kills Inactive Server
```

Comment:
does `Modify Culler Timeout` takes much time? if not maybe we can make the TC indipendet and re-call here the kw to change the timeout to some convinient value

Reply:
No, I don't think it takes more than a couple minutes. But when this TC runs the culler is already set at lowest possible value (10 minutes)

Reply:
yeah the thing here is the question about independe vs dependent TCs. This one depends on the previous one which sets the new timeout. A discussion which never finds a conclusion eheh. I don't know, if keeping these dependent makes us spare execution time, I guess it could be ok for me..

Reply:
In this context I'm not sure how much it would make sense, since I don't think we would run any of these TCs in isolation?
We can discuss about it more and implement it as a future improvement though

Reply:
yeah let's keep it as point for automation meeting

Reply:
I also think tests should not depend on each other, specially if there is not a big time penalty to make them independent.

Another option is at least to add a tag or something to note this dependency. I just added a proposed tag for this in the style guide